Dockerfile

```diff
@@ -0,0 +1,30 @@
+# A Docker container for https://github.com/skibish/ddns
```

Comment:
In my opinion this line can be removed, because this file already located in the project (so, it should be used here)